samples/bluetooth/llpm/README.rst

```diff
@@ -77,12 +77,16 @@ Requirements
 
 * Two of the following nRF52-series development kit boards:
 
+  * |nRF5340DK|
   * |nRF52DK|
   * |nRF52840DK|
   * Other boards running BLE Controller variants that support LLPM (see :ref:`nrfxlib:ble_controller` Proprietary feature support)
 
   You can mix different boards.
 * Connection to a computer with a serial terminal for each of the boards.
+* nRF Bluetooth LE Controller.
+  This is enabled by default, but it does not currently propagate correctly for nRF5340.
+  It must be enabled in the :file:`.config` file - see :ref:`configure_application`.
```

Comment:
What option(s) do you need to set?

Reply:
With https://github.com/nrfconnect/sdk-nrf/pull/2467, it will work correctly for nRF5340, so I will remove this.